### ISSUE TITLE
Breaking change in dmmf coming in 2.29 release - `idFields` removed, replaced by `primaryKey`

### ISSUE THREAD

**Issue by Jolg42** [notiz-dev/prisma-dbml-generator#17]
Just noticed that a change needed for 2.29 release will affect your project, so I wanted to notify you here to make it easier to upgrade.

*Note: the DMMF is not part of the public API so this is not an "official" breaking change and Prisma adopting semver releases soon will not apply here as well.*

The test where I noticed an error:
https://github.com/prisma/e2e-tests/runs/3240419049?check_suite_focus=true#step:5:64

The source code where this is coming I think:
https://github.com/notiz-dev/prisma-dbml-generator/blob/b32bcfc2f5b31762cb4da9e30fec59eb14bfec06/src/generator/table.ts#L16

The change:
```ts
// Model type
idFields: string[]
```

is now replaced by
```ts
export interface PrimaryKey {
  name: string | null
  fields: string[]
}

// Model type
primaryKey: PrimaryKey | null 
```

For more context see Prisma Client PR https://github.com/prisma/prisma/pull/8562/files

**Comment by marcjulian** [comment]
Fixed in 8ac5f75

**Comment by marcjulian** [comment]
Released a new version [0.7.0](https://github.com/notiz-dev/prisma-dbml-generator/releases/tag/v0.7.0) to support Prisma version 2.29.0.

```bash
npm i -D prisma-dbml-generator@latest
```
